crates/stackable-telemetry/src/tracing/mod.rs

```diff
@@ -244,12 +280,70 @@ pub struct Tracing {
 }
 
 impl Tracing {
+    /// The environment variable used to set the console log level filter.
+    pub const CONSOLE_LOG_ENV_VAR: &str = "CONSOLE_LOG";
+    /// The environment variable used to set the rolling file log level filter.
+    pub const FILE_LOG_ENV_VAR: &str = "FILE_LOG";
+    /// The filename used for the rolling file logs.
+    pub const FILE_LOG_SUFFIX: &str = "tracing-rs.json";
+    /// The environment variable used to set the OTLP log level filter.
+    pub const OTLP_LOG_ENV_VAR: &str = "OTLP_LOG";
+    /// The environment variable used to set the OTLP trace level filter.
+    pub const OTLP_TRACE_ENV_VAR: &str = "OTLP_TRACE";
+
     /// Creates and returns a [`TracingBuilder`].
     pub fn builder() -> TracingBuilder<builder_state::PreServiceName> {
         TracingBuilder::default()
     }
 
-    /// Initialise the configured tracing subscribers, returning a guard that
+    /// Creates an returns a pre-configured [`Tracing`] instance which can be initialized by
+    /// calling [`Tracing::init()`].
+    ///
+    /// ### Environment Variables and Default Levels
+    ///
+    /// | Level Filter for | Environment Variable                       | Default Level |
+    /// | ---------------- | ------------------------------------------ | ------------- |
+    /// | Console logs     | [`CONSOLE_LOG`](Self::CONSOLE_LOG_ENV_VAR) | `INFO`        |
+    /// | File logs        | [`FILE_LOG`](Self::FILE_LOG_ENV_VAR)       | `INFO`        |
+    /// | OTLP logs        | [`OTLP_LOG`](Self::OTLP_LOG_ENV_VAR)       | `DEBUG`       |
+    /// | OTLP traces      | [`OTLP_TRACE`](Self::OTLP_TRACE_ENV_VAR)   | `DEBUG`       |
```

Comment:
I just realised, for production we probably want to default to `INFO` and require explicit setting for deeper level info.

You can still see spans in tracing (unless they are demoted to `DEBUG` or `TRACE` level).

_From memory, the default level for `instrument` spans is `INFO` - needs checking_

Reply:
```suggestion
    /// | OTLP logs        | [`OTLP_LOG`](Self::OTLP_LOG_ENV_VAR)       | `INFO`       |
    /// | OTLP traces      | [`OTLP_TRACE`](Self::OTLP_TRACE_ENV_VAR)   | `INFO`       |
```